static/.htaccess

```diff
@@ -1,16 +0,0 @@
-# Set far-future Expires headers for static media
-ExpiresActive on
-
-ExpiresDefault "access plus 1 week"
-ExpiresByType text/css "access plus 1 year"
-ExpiresByType text/javascript "access plus 1 year"
-ExpiresByType image/png "access plus 1 month"
-ExpiresByType image/jpeg "access plus 1 month"
-ExpiresByType image/gif "access plus 1 month"
-ExpiresByType image/vnd.microsoft.icon "access plus 1 month"
-ExpiresByType video/webm "access plus 1 week"
-ExpiresByType video/ogg "access plus 1 week"
-ExpiresByType video/x-flv "access plus 1 week"
-ExpiresByType application/x-shockwave-flash "access plus 1 week"
```

Comment:
Whitenoise will by default use an expiration time of 60 seconds for non-hashed files: https://github.com/evansd/whitenoise/blob/0657e7aea986fa652bd04d6ae8468a0e1129d634/whitenoise/base.py#L89

For hashed files ("immutable files") it'll use a far future value: https://github.com/evansd/whitenoise/blob/0657e7aea986fa652bd04d6ae8468a0e1129d634/whitenoise/base.py#L229-L230

So I think we should be good since in production we'll ever only use hashed values and for the non-hashed files we can allow the CDN to fetch the files every 60 seconds IMO.